components/e2c/ResourcesSection.js

```diff
@@ -83,12 +84,12 @@ const ResourcesSection = () => {
           controlsTopPosition={30}
           containerProps={{ hideScrollbar: true }}
         >
-          {blogEntries.map((blogEntry, index) => (
+          {blogEntries.map(blogEntry => (
             <Container
               display="flex"
               flexDirection="column"
               minWidth={['288px', '276px', '380px', null, '360px']}
-              key={index.toString()}
+              key={uuidv4()}
```

Comment:
What's the reason behind this change? Can you describe how React is going to handle the `key` set to a random UUID?